test/parallel/test-fs-readdir-ucs2.js

```diff
@@ -16,16 +16,20 @@ const root = Buffer.from(`${common.tmpDir}${path.sep}`);
 const filebuff = Buffer.from(filename, 'ucs2');
 const fullpath = Buffer.concat([root, filebuff]);
 
-fs.closeSync(fs.openSync(fullpath, 'w+'));
+try {
+  fs.closeSync(fs.openSync(fullpath, 'w+'));
+} catch (e) {
+  if (e.message.startsWith('EINVAL')) {
```

Comment:
Is `e.code` set? If so it might be better to test against it rather than the message.